receipt_chroma/receipt_chroma/s3/helpers.py

```
@@ -60,6 +106,299 @@ class DeltaTarballResult(TypedDict, total=False):
+            else:
+                return (s3_key, 0, f"Error after {MAX_DOWNLOAD_RETRIES} retries: {e}")
+
+    return (s3_key, 0, f"Failed after {MAX_DOWNLOAD_RETRIES} retries: {last_error}")
+
+
+def download_snapshot_from_s3_parallel(
+    bucket: str,
+    snapshot_key: str,
+    local_snapshot_path: str,
+    max_workers: int = 8,
+    verify_integrity: bool = False,
```

Ruff (0.14.10)

Unused function argument: `verify_integrity`

(ARG001)